custom-completions/docker/docker-completions.nu

```diff
@@ -514,6 +519,54 @@ export extern "docker buildx" [
     --builder: string                                   #Override the configured builder instance (default "default")
 ]
 
+# Run docker compose using docker-compose.yml
+export extern "docker compose up" [
+    --abort-on-container-exit                           #Stops all containers if any container was stopped. Incompatible with -d/--detach
+    --abort-on-container-failure                        #Stops all containers if any container had a non-zero exit code. Incompatible with -d/--detach
+    --always-recreate-deps                              #Recreate dependent containers. Incompatible with --no-recreate
+    --attach: list<string>                              #Restrict attaching to the specified services. Incompatible with --attach-dependencies
+    --attach-dependencies                               #Automatically attach to log output of all dependent services
+    --build                                             #Build images before starting containers
+
```

Comment:
I'm not sure if spaces in this list here is a good idea. I wonder if it could cause problems when using it.

Reply:
I tried to use `...attach: string`. But this way `attach` is no longer a flag
`--attach: ...string` has wrong grammar.
Little confused now